Dockerfile_with_migrate

```diff
@@ -0,0 +1,35 @@
+FROM node:20.12-alpine
```

Comment:
Should this have a different name like `Dockerfile.cloudrun` to make it clearer what it's for?

Reply:
@rhelmer would this be better? I think maybe we can just reuse the one Dockerfile: https://github.com/mozilla/blurts-server/pull/4448/commits/ecc959aeae0a4b3904ca11fd939fdff8f49adfdc

Reply:
nvm that didn't work, trying: https://github.com/mozilla/blurts-server/pull/4448/commits/51ed1c1ed2f6878fdcff245e400dbc36167bab5d

Reply:
Hmm, ok so I'm not able to get the same dockerfile to work with conditionals. I think it has to do with the context when `docker build` happens and when `docker run` happens are in different places (here in github, then in cloudrun).. So the only way is to have different `CMD`

Reply:
But I did rename to `Dockerfile.cloudrun`: https://github.com/mozilla/blurts-server/pull/4448/commits/4daec19ed1ad6e777e92d43df1bd53cb8473623e